packages/next/src/server/app-render/work-unit-async-storage.external.ts

```diff
@@ -56,16 +56,16 @@ export type RequestStore = {
   /**
    * The resume data cache for this request. This will be a immutable cache.
    */
-  immutableResumeDataCache: ImmutableResumeDataCache | null
-
-  /**
-   * The resume data cache for this request. This will be a mutable cache.
-   */
-  mutableResumeDataCache: MutableResumeDataCache | null
+  renderResumeDataCache: RenderResumeDataCache | null
 
   // DEV-only
   usedDynamic?: boolean
   prerenderPhase?: boolean
+
+  /**
+   * The resume data cache for this request. This will be a mutable cache.
+   */
+  devWarmupPrerenderResumeDataCache: PrerenderResumeDataCache | null
```

Comment:
definitely keep this in but a heads up I'm going to be changing the warmup pass to be scoped with a prerender store so this part of the requestStore type will be going away anyway which is kinda nice